github/github_repo.go

```diff
@@ -66,6 +71,19 @@ func (repo *GithubRepo) oauthClient() *http.Client {
 	return oauth2.NewClient(context.Background(), tokenService)
 }
 
+func (repo *GithubRepo) githubClient() (*ghb.Client, error) {
+	oauthClient := repo.oauthClient()
+
+	if len(repo.baseURL) > 0 {
+		if len(repo.uploadURL) == 0 {
+			repo.uploadURL = repo.baseURL
```

Comment:
Where is `uploadURL` used? In the next line, `baseURL` is in there twice. I assume one should be `uploadURL`?